<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="header" native="true">
     <property name="maximumHeight">
      <number>43</number>
     </property>
     <property name="minimumHeight">
      <number>43</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string>QApp</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="avatarButton">
        <property name="text">
         <string>Avatar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QWidget" name="profileDetailsWidget">
     <layout class="QFormLayout" name="profileDetailsLayout">

      <!-- UserName -->
      <item row="0" column="0">
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>UserName</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="userNameLineEdit"/>
      </item>

      <!-- Email -->
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>

      <!-- Dob -->
      <item row="2" column="0">
       <widget class="QLabel" name="dobLabel">
        <property name="text">
         <string>Date of Birth</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dobLineEdit"/>
      </item>

      <!-- Password -->
      <item row="3" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="passwordFieldLayout">
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="togglePasswordButton">
          <property name="text">
           <string>üëÅÔ∏è</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>

      <!-- Save Button -->
      <item row="4" column="1" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
